AU/Public/Update-Package.ps1

```diff
@@ -129,12 +129,13 @@ function Update-Package {
             $Env:ChocolateyPackageVersion      = $global:Latest.Version.ToString()
             $Env:ChocolateyAllowEmptyChecksums = 'true'
             foreach ($a in $arch) {
+                "Checking hash for $a bit version..." | result
                 $Env:chocolateyForceX86 = if ($a -eq '32') { 'true' } else { '' }
                 try {
                     #rm -force -recurse -ea ignore $pkg_path
                     .\tools\chocolateyInstall.ps1 | result
                 } catch {
-                    if ( "$_" -notlike 'au_break: *') { throw $_ } else {
+                    if ( "$_" -notlike 'au_break: *') { throw } else {
```

Comment:
Why are you removing the error message, here?

Reply:
I'm not, this is a specific syntax to rethrow the caught exception (instead of throwing a new exception based on the current one).

Reply:
I've been throwing the exception for a long time without realising this - thank you. TIL.